app/renderer/js/pages/preference/find-accounts.ts

```diff
@@ -1,16 +1,21 @@
 'use-strict';
 
-const { shell } = require('electron');
+import { shell } from 'electron';
 
-const BaseComponent = require(__dirname + '/../../components/base.js');
+import BaseComponent = require('../../components/base');
```

Comment:
This path will be automatically resolved on Windows, right?

Reply:
Yes, see https://github.com/zulip/zulip-desktop/pull/782/commits/8b7953343ded2e2a04218856ff42299283fe5bc7 for more details.

Reply:
Got it, good commit message btw.